### DISCUSSION TITLE
Feeling kind of silly but...

### DISCUSSION THREAD

**Discussion by DraqueB** [astrandb/miele#154] [Q&A]
i cant get the repo into hacs.
This is the error i get:

https://github.com/astrandb/miele/ is not a valid add-on repository

Sorry, looks like i am an absolute beginner.

**Comment by rudizl** [comment]
https://github.com/astrandb/miele.git

**Reply by DraqueB** [reply]
Still the same: https://github.com/astrandb/miele.git is not a valid add-on repository

**Reply by rudizl** [reply]
![image](https://user-images.githubusercontent.com/5582186/200623226-8a4344bd-9ab0-42f2-9999-98f3894218fa.png)
And you`re following those steps?

**Reply by DraqueB** [reply]
Thanks for your patience, as i thought a made a silly mistake.
